### ISSUE TITLE
"restart node" handler not found

### ISSUE THREAD

**Issue by vishpat** [openshift/openshift-ansible#1299]
Setup is similar to issue #1298 , the node setup for nuage is failing since it is unable to find "restart node" handler.

2016-01-28 14:49:09,000 p=28536 u=ovs-7 |  TASK: [nuage_node | Set the vsp-k8s.yaml] *************************************
2016-01-28 14:49:09,056 p=28536 u=ovs-7 |  <node1.nuageopenshift.com> ESTABLISH CONNECTION FOR USER: root
2016-01-28 14:49:09,057 p=28536 u=ovs-7 |  <node1.nuageopenshift.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node1.nuageopenshift.com /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877 && chmod a+rx $HOME/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877 && echo $HOME/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877'
2016-01-28 14:49:09,059 p=28536 u=ovs-7 |  <node2.nuageopenshift.com> ESTABLISH CONNECTION FOR USER: root
2016-01-28 14:49:09,060 p=28536 u=ovs-7 |  <node2.nuageopenshift.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node2.nuageopenshift.com /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161 && chmod a+rx $HOME/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161 && echo $HOME/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161'
2016-01-28 14:49:09,139 p=28536 u=ovs-7 |  <node2.nuageopenshift.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node2.nuageopenshift.com /bin/sh -c 'rc=flag; [ -r /usr/share/vsp-k8s/vsp-k8s.yaml ] || rc=2; [ -f /usr/share/vsp-k8s/vsp-k8s.yaml ] || rc=1; [ -d /usr/share/vsp-k8s/vsp-k8s.yaml ] && rc=3; python -V 2>/dev/null || rc=4; [ x"$rc" != "xflag" ] && echo "${rc} "/usr/share/vsp-k8s/vsp-k8s.yaml && exit 0; (python -c '"'"'import hashlib; BLOCKSIZE = 65536; hasher = hashlib.sha1(); afile = open("'"'"'/usr/share/vsp-k8s/vsp-k8s.yaml'"'"'", "rb") buf = afile.read(BLOCKSIZE) while len(buf) > 0: hasher.update(buf) buf = afile.read(BLOCKSIZE) afile.close() print(hasher.hexdigest())'"'"' 2>/dev/null) || (python -c '"'"'import sha; BLOCKSIZE = 65536; hasher = sha.sha(); afile = open("'"'"'/usr/share/vsp-k8s/vsp-k8s.yaml'"'"'", "rb") buf = afile.read(BLOCKSIZE) while len(buf) > 0: hasher.update(buf) buf = afile.read(BLOCKSIZE) afile.close() print(hasher.hexdigest())'"'"' 2>/dev/null) || (echo '"'"'0 '"'"'/usr/share/vsp-k8s/vsp-k8s.yaml)'
2016-01-28 14:49:09,143 p=28536 u=ovs-7 |  <node1.nuageopenshift.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node1.nuageopenshift.com /bin/sh -c 'rc=flag; [ -r /usr/share/vsp-k8s/vsp-k8s.yaml ] || rc=2; [ -f /usr/share/vsp-k8s/vsp-k8s.yaml ] || rc=1; [ -d /usr/share/vsp-k8s/vsp-k8s.yaml ] && rc=3; python -V 2>/dev/null || rc=4; [ x"$rc" != "xflag" ] && echo "${rc} "/usr/share/vsp-k8s/vsp-k8s.yaml && exit 0; (python -c '"'"'import hashlib; BLOCKSIZE = 65536; hasher = hashlib.sha1(); afile = open("'"'"'/usr/share/vsp-k8s/vsp-k8s.yaml'"'"'", "rb") buf = afile.read(BLOCKSIZE) while len(buf) > 0: hasher.update(buf) buf = afile.read(BLOCKSIZE) afile.close() print(hasher.hexdigest())'"'"' 2>/dev/null) || (python -c '"'"'import sha; BLOCKSIZE = 65536; hasher = sha.sha(); afile = open("'"'"'/usr/share/vsp-k8s/vsp-k8s.yaml'"'"'", "rb") buf = afile.read(BLOCKSIZE) while len(buf) > 0: hasher.update(buf) buf = afile.read(BLOCKSIZE) afile.close() print(hasher.hexdigest())'"'"' 2>/dev/null) || (echo '"'"'0 '"'"'/usr/share/vsp-k8s/vsp-k8s.yaml)'
2016-01-28 14:49:09,227 p=28536 u=ovs-7 |  <node2.nuageopenshift.com> PUT /tmp/tmpwhkViH TO /root/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161/source
2016-01-28 14:49:09,255 p=28536 u=ovs-7 |  <node1.nuageopenshift.com> PUT /tmp/tmpey2RTC TO /root/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877/source
2016-01-28 14:49:09,282 p=28536 u=ovs-7 |  <node2.nuageopenshift.com> EXEC ssh -C -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node2.nuageopenshift.com /bin/sh -c 'sudo -k && sudo -H -S -p "[sudo via ansible, key=gotjajnkxeypoyhyhgaziuywmjmwamxm] password: " -u root /bin/sh -c '"'"'echo BECOME-SUCCESS-gotjajnkxeypoyhyhgaziuywmjmwamxm; LANG=en_US.UTF-8 LC_CTYPE=en_US.UTF-8 /usr/bin/python ; rm -rf /root/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161/ >/dev/null 2>&1'"'"''
2016-01-28 14:49:09,317 p=28536 u=ovs-7 |  <node1.nuageopenshift.com> EXEC ssh -C -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/home/ovs-7/.ansible/cp/ansible-ssh-%h-%p-%r" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 node1.nuageopenshift.com /bin/sh -c 'sudo -k && sudo -H -S -p "[sudo via ansible, key=vgionkrdxjnqstzdkxgdcqjgeyufhvkx] password: " -u root /bin/sh -c '"'"'echo BECOME-SUCCESS-vgionkrdxjnqstzdkxgdcqjgeyufhvkx; LANG=en_US.UTF-8 LC_CTYPE=en_US.UTF-8 /usr/bin/python ; rm -rf /root/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877/ >/dev/null 2>&1'"'"''
2016-01-28 14:49:09,453 p=28536 u=ovs-7 |  changed: [node2.nuageopenshift.com] => {"changed": true, "checksum": "e9692a3155295da52afc9f8600302ef2f2c5590e", "dest": "/usr/share/vsp-k8s/vsp-k8s.yaml", "gid": 99, "group": "nobody", "md5sum": "aba6b8e410d6e38a8f107dba6c1d1ac5", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:usr_t:s0", "size": 615, "src": "/root/.ansible/tmp/ansible-tmp-1454021349.06-121868817980161/source", "state": "file", "uid": 0}
2016-01-28 14:49:09,537 p=28536 u=ovs-7 |  changed: [node1.nuageopenshift.com] => {"changed": true, "checksum": "e9692a3155295da52afc9f8600302ef2f2c5590e", "dest": "/usr/share/vsp-k8s/vsp-k8s.yaml", "gid": 99, "group": "nobody", "md5sum": "aba6b8e410d6e38a8f107dba6c1d1ac5", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:usr_t:s0", "size": 615, "src": "/root/.ansible/tmp/ansible-tmp-1454021349.06-96272827003877/source", "state": "file", "uid": 0}
2016-01-28 14:49:09,558 p=28536 u=ovs-7 |  ERROR: change handler (restart node) is not defined

**Comment by vishpat** [comment]
@abutcher @detiber Any suggestions how do I fix this

**Comment by abutcher** [comment]
The `nuage_node` role needs the handler defined.

```
diff --git a/roles/nuage_node/handlers/main.yaml b/roles/nuage_node/handlers/main.yaml
index d82d4b6..3fd1da5 100644
--- a/roles/nuage_node/handlers/main.yaml
+++ b/roles/nuage_node/handlers/main.yaml
@@ -2,3 +2,6 @@
 - name: restart vrs
   sudo: true
   service: name=openvswitch state=restarted
+
+- name: restart node
+  service: name={{ openshift.common.service_type }}-node state=restarted
```

**Comment by vishpat** [comment]
@abutcher Will make the change and send it for review

**Comment by abutcher** [comment]
@brenton This one can be closed. Resolved by #1301
